tests/integration_tests/performance/vsock_throughput_test_config.py

```diff
@@ -0,0 +1,320 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Configuration file for the VSOCK throughput test."""
+
+IPERF3 = "iperf3-vsock"
+THROUGHPUT = "throughput"
+THROUGHPUT_TOTAL = "total"
+DURATION = "duration"
+DURATION_TOTAL = "total"
+CPU_UTILIZATION_HOST = "cpu_utilization_host"
+CPU_UTILIZATION_GUEST = "cpu_utilization_guest"
+BASE_PORT = 5201
+CPU_UTILIZATION_VMM_TAG = "vmm"
+CPU_UTILIZATION_VCPUS_TOTAL_TAG = "vcpus_total"
+IPERF3_CPU_UTILIZATION_PERCENT_OUT_TAG = "cpu_utilization_percent"
+IPERF3_END_RESULTS_TAG = "end"
+TARGET_TAG = "target"
+DELTA_PERCENTAGE_TAG = "delta_percentage"
+THROUGHPUT_UNIT = "Mbps"
+DURATION_UNIT = "seconds"
+CPU_UTILIZATION_UNIT = "percentage"
+
+
+CONFIG = {
+    "time": 20,  # seconds
+    "server_startup_time": 2,  # seconds,
+    "load_factor": 1,  # nr of iperf clients/vcpu
+    "modes": {
+        "g2h": [""],
+        "h2g": ["-R"],
+        "bd": ["", "-R"]
+    },
+    "protocols": [
+        {
+            "name": "tcp",
+            "omit": 3,  # seconds
+            "payload_length": ["1024K",  None],  # None == Default
```

Comment:
I would rather put a smaller payload than the default one. 1024K is a really high one, maybe try with 1024? This can simulate a microservice for control plane operations for example where you just exchange some json  messages.

Reply:
I tried adding a very small payload: `96` bytes. However, the cpu utilization results had a very high standard deviation. The delta percentage was more than 50%, therefore would bring no benefit

Reply:
still I don't see a benefit in 1024k payload vs 128k. Maybe give a try with 1024.

Reply:
Maybe the 96 bytes payload was too small, I could give it a try with 1024 bytes 10KB or and check out the results

Reply:
Please try 1024 first.

Reply:
will do

Reply:
looks like the 1024B payload makes for a reasonable delta percentage. will gather baselines and update the PR with this new length

Reply:
Done. Added the baselines for 1024B